packages/pwa/app/constants.js

```diff
@@ -57,3 +57,8 @@ export const SUPPORTED_LOCALES = packageInfo.l10n.supportedLocales
 export const DEFAULT_LOCALE = packageInfo.l10n.defaultLocale
 export const SUPPORTED_CURRENCIES = packageInfo.l10n.supportedCurrencies
 export const DEFAULT_CURRENCY = packageInfo.l10n.defaultCurrency
+
+export const urlParamTypes = {
```

Comment:
A couple comments on this enum: 

1. We'll probably need a value for `NONE: 'none'` in the situations where the implementer doesn't want to display the parameter in the url at all. Scenarios would be single site, single locale apps. In that case (which is going to be out default) we will want to know how to place those params.
2. I'm not sold on the name, specifically the word "Params" in urlParamTypes. This enum should be talking about placement or positions of config values in the url. A name I can think of that is totally obvious (maybe too obvious) is `urlPositions`.

Reply:
I find `urlPositions` is a bit too generic, when we say url, it implicates the whole path (e.g en-US/men/suits/) or just a subset of it. In fact, we only want to address a part of it (/en-US/) one at a time. The word `Params`(though this should be a single noun instead of plural) still makes sense to me because regardless of the position, they are still the parameters (url parameters vs url query parameters or none) as they are dynamic values. Maybe we can call it `urlParamPositions`?

Reply:
we had a chat on the naming, and find that using `urlPartPositions` makes the most sense.